atlasdb-client/src/main/java/com/palantir/atlasdb/table/description/Schema.java

```diff
@@ -68,6 +68,9 @@
     private final String packageName;
     private final Namespace namespace;
     private final OptionalType optionalType;
+    private boolean ignoreTableNameLengthChecks = false;
+    private static final int CASSANDRA_TABLE_NAME_CHAR_LIMIT = 48;
```

Comment:
Constants generally live in `CassandraConstants`, `AtlasDBConstants` etc, unless they are specific to the class..